### PR TITLE
Only put CJK chunks in span (#31)

### PR DESCRIPTION
**Pull request by adamyi**
This pull request addresses issue #31.

```
>>> import budou
>>> parser = budou.authenticate('/Users/adamyi/credential.json')
>>> result = parser.parse(u'Let us test this fun 今日も元気です tool!', attributes={'class': 'wordwrap'}, language='ja')
>>> print(result['html_code'])
<span>Let us test this fun <span class="wordwrap">今日も</span><span class="wordwrap">元気です </span>tool!</span>
```

### CODE REVIEW

**Comment by adamyi** [comment]
My bad... There appears to be a problem with the unit test... I'll look into that.

**Comment by adamyi** [comment]
It turns out that there was a test case for `<span>a</span><span>b</span><span>c</span>`, but they should not be added in the span tags since they are English. I changed the test case to a new one with both CJK chars and English chars to reflect this new logic.

File: budou/budou.py [hunk 1]
```diff
@@ -67,6 +67,19 @@ def is_space(self):
     """Checks if this is space Chunk."""
     return self.pos == self.SPACE_POS
 
+  def has_cjk(self):
+    """Checks if this contains CJK characters"""
```
Comment (tushuhei) [inline budou/budou.py]: Could you put a reference where you got the range of Unicode code points for CJK in this comment?

File: budou/budou.py [hunk 2]
```diff
@@ -67,6 +67,19 @@ def is_space(self):
...
+    i = 0
+    while i < len(self.word):
```
Comment (tushuhei) [inline budou/budou.py]: I think `for w < self.word:` also works and it's more straightforward.

File: budou/budou.py (hunk 1 above)
Comment (adamyi) [reply budou/budou.py]: On a StackOverflow post, which refers to https://github.com/nltk/nltk/blob/develop/nltk/tokenize/util.py#L149

**Comment by tushuhei** [comment]
LGTM. Thank you!

**Pull request merged**